apps/client/src/main.tsx

```diff
@@ -8,7 +10,9 @@ const root = ReactDOM.createRoot(
 );
 
 root.render(
-  <StrictMode>
-    <App />
-  </StrictMode>,
+  <Provider store={store}>
+    <StrictMode>
```

Comment:
Why is `Provider` outside of `StrictMode`?

Reply:
Changed 95f0ac22e69d8ae243c63c4cd59f9fd38b67ab91